people.php

```diff
@@ -0,0 +1,152 @@
+<!DOCTYPE html>
+<html>
+
+<head>
+	<?php
+	$conn = require_once (__DIR__ . "/scripts/connection.php");
+	if (isset($_GET['id'])) {
+		$id = $_GET['id'];
+	} else {
+		# echo ("no person selected");
+		echo "
+					<title>People</title>
+					<link rel='icon' href='./img/default_icon.png'>
+					<link rel='stylesheet' href='./css/fonts.css'>
+					<link rel='stylesheet' href='/css/menu_people.css'>
+					";
+		require (__DIR__ . "/scripts/menus.php");
+		peopleMenu();
+		exit();
+	}
+
+	// Query
+	$query = $conn->prepare("SELECT * FROM people where id=?");
+	$query->bind_param('i', $id);
+	$query->execute();
+	$result = $query->get_result()->fetch_assoc();
+
+	if (empty($result)) {
+		echo "Does not exist";
+		echo "<title>People</title>";
+		exit();
+	}
+
+	?>
+	<title><?= $result['name'] ?> - People</title>
+<meta charset="utf-8">
+	<?php
+	if (file_exists("./img/$result[name]_icon.png")) {
+		echo "<link rel='icon' href='./img/$result[name]_icon.png'>";
+	}
+	?>
+	<link rel="stylesheet" href="./css/fonts.css">
+	<link rel="stylesheet" href="./css/people.css">
+	<link rel="stylesheet" href="./css/peopleSmall.css" media="all and (max-aspect-ratio: 3/5)">
+	<script src="/scripts/people_styling.js"></script>
+</head>
+
+<body>
+	<a href="people.php">Back to TOC</a>
+	<table role="main">
+		<caption>
+			<h1><?=$result['name']?></h1>
+		</caption>
+
+		<?php
+		$imgpath = "./img/$result[name].jpg";
+		if (file_exists($imgpath)) {
+			echo "<link rel='stylesheet' href='./css/rowFix.css'>";
+			echo "
+			                <tr>
+			                    <td><img src='$imgpath'></img></td>
+			                </tr>
+			                ";
+		}
+		?>
+		<tr>
+			<th>Gender</th>
+			<td><?php
+			if ($result['gender'] == 'f') {
+				echo 'Female';
+			} elseif ($result['gender'] == 'm') {
+				echo 'Male';
+			} elseif ($result['gender'] == 'd') {
+				echo 'Non-Binary';
+			}
+
+			?>
+			</td>
+		</tr>
+
+		<?php
+		if ((!empty($result['birth-day'])) && (!empty($result['birth-year']))) {
+			echo ("<tr>
+			                <th>Birthday</th>
+			                <td>" . $result['birth-day'] . ". " . $result['birth-year'] . "</td>
+					</tr>");
+		}
+		?>
+		<?php
+		if (!empty($result['weight'])) {
+			echo ("<tr>
+			                <th>Weight</th>
+			                <td>$result[weight] kg</td>
```

Comment:
```suggestion
			                <td>$result[weight]&nbsp;kg</td>
```

Reply:
There's supposed to be a space there

Reply:
Do you want the unit to break to a new line?

Reply:
(Although since this is a table cell with just one number in it, that's probably an unrealistic edge case, so the non breaking space is actually not necessary here)